scripts/docker/README

```diff
@@ -0,0 +1,37 @@
+Dockerfiles are used to create Docker images.  Docker images are the 'recipes' used to build virtual environments called containers.
+These dockerfiles all use a base Linux image that has a particular compiler installed.  These base images were created by David
+Beckinsale and are located on Docker Hub at this location:
+https://hub.docker.com/r/axom/compilers/tags
+
+The base images are accessed in these dockerfiles by the use of the 'FROM' command:
+FROM axom/compilers:<compiler>
+
+The remaining dockerfile commands attempt to build the Axom third party libraries using the specified compiler.
+The following commands update the compiler and add Fortran compiler support:
```

Comment:
@davidbeckingsale -- should we fold these commands into the compiler docker images that way we do not need to rerun them each time?

(or am I reading this wrong?)